framework/src/main/java/org/checkerframework/common/wholeprograminference/scenelib/AMethodWrapper.java

```diff
@@ -72,12 +64,12 @@
      *
      * @param methodElt the method whose parameters should be vivified
```

Comment:
The documentation here needs to be updated to no longer use the word "vivify", because the name has been changed.

That said, an important side-effect of this method is that it does call `vivifyParameter` on every parameter in the underlying `AMethod`. If it did not do that, then some method signatures in stub files would be printed with only some of their parameters (the annotated ones), which is obviously not valid in a stub file.